pycross/private/lock_attrs.bzl

```diff
@@ -92,7 +92,7 @@ RENDER_ATTRS = dict(
     ),
 ) | CREATE_REPOS_ATTRS
 
-PDM_IMPORT_ATTRS = dict(
+_IMPORT_ATTRS = dict(
     lock_file = attr.label(
         doc = "The pdm.lock file.",
```

Comment:
This comment isn't accurate when used with uv. Options are to extract some common attributes and append PDM- and uv-specific `lock_file` entries, or to just change the doc to not specify a filename.